.github/workflows/subgraph-documentation-update.yml

```diff
@@ -15,7 +15,7 @@ on:
       - '.github/workflows/subgraph-deploy.yaml'
```

Comment:
Maybe the problem is here. I'm not sure gh will pick up changes in examples if wue dont add `/**`
```yml
on:
  workflow_dispatch:
  workflow_call:
  push:
    branches:
      - develop
    paths:
      - 'packages/subgraph/schema.graphql'
      - 'packages/subgraph/src/examples/**'
      - '.github/workflows/subgraph-deploy.yaml'
   ```

Reply:
I think adding /** will be necessary to take into account subdirectories too. Which we may need it, I'll add it.